powerline/segments/common/wthr.py

```diff
@@ -105,6 +105,7 @@ class WeatherSegment(KwThreadedSegment):
 	interval = 600
 	default_location = None
 	location_urls = {}
+	weather_api_key = "fbc9549d91a5e4b26c15be0dbdac3460"
```

Comment:
Perhaps I'm missing something here, but it seems wrong to me to have an API key hard-coded.

Reply:
that could well be in the config file :+1

Reply:
Yeah, I will  to make that work.

Reply:
How about adding the api key as a parameter to the segment?